python/cudf/cudf/core/dataframe.py

```diff
@@ -5861,7 +5861,49 @@ def _reduce(
                     for col in source._data.names
                 ]
             except AttributeError:
-                raise TypeError(f"Not all column dtypes support op {op}")
+                numeric_ops = (
+                    "mean",
+                    "min",
+                    "max",
+                    "sum",
+                    "product",
+                    "prod",
+                    "std",
+                    "var",
+                    "kurtosis",
+                    "kurt",
+                    "skew",
+                )
+
+                if numeric_only is None and op in numeric_ops:
+                    warnings.warn(
+                        "The default value of numeric_only in DataFrame.kurt "
```

Comment:
Is this warning just for `DataFrame.kurt`?

Reply:
Nope, thanks. Fixed.